### DISCUSSION TITLE
Is there a way to deep link to a email message by id

### DISCUSSION THREAD

**Discussion by cameronbraid** [postalserver/postal#2095] [Help with using Postal]
I would like to link from my app to the postal message view.  Is there a way to way to generate a link using the mime message id header ?  For example to the url /org/ORG/servers/SERVER/messages/X X looks to be an internal id.    If this isn't supported some other way - could the controller be changed so that if X is not a number that it looks up the message by message id header ?

**Comment by willpower232** [comment]
When you send messages with the API you get the numeric ID of the email you've sent which I believe is the ID which is in the URL in the Postal UI.

FWIW this is also the ID you would need to use the messages API so worth keeping in your database for debugging reasons.

**Comment by cameronbraid** [comment]
I send emails via smtp so that data is not available

Cam

________________________________

Cameron Braid
Director
E:
T:
 ***@***.***
 1300 547 214
  61 3 9095 7460
M:
W:
  61 438 752 513
 www.drivenow.com.au
Helping travellers find great deals on rental vehicles since 2003
Australia
The information transmitted is intended only for the person or entity to which it is addressed and may contain confidential and/or privileged material. Any use, dissemination, distribution, reproduction, or taking of any action in reliance upon, this information by persons or entities other than the intended recipient is prohibited. If you received this transmission in error, please contact the sender by reply e-mail and delete and destroy all copies of the material. 
DriveNow has taken precautions to minimise the risk of transmitting software viruses, but we advise you to carry out your own virus checks on any attachment to this message. DriveNow accepts no liability for any loss or damage caused by software viruses or malware whilst opening or reading this message and its enclosures.
